noxfile.py

```diff
@@ -63,12 +63,6 @@ def should_update_common_wheels() -> bool:
     return need_to_repopulate
 
 
-# -----------------------------------------------------------------------------
-# Development Commands
-#   These are currently prototypes to evaluate whether we want to switch over
-#   completely to nox for all our automation. Contributors should prefer using
-#   `tox -e ...` until this note is removed.
-# -----------------------------------------------------------------------------
```

Comment:
I removed this whole comment (because `tox` won't be accesible now). But maybe I should keep the `Development Commands` part?